system-test/compute.js

```diff
@@ -155,6 +188,9 @@ describe('Compute', () => {
     });
 
     it('create and fetch firewall, test capital letter field like "IPProtocol"', async function () {
+      this.retries(3);
+      await delay(this.test);
```

Comment:
For learning purpose, what is `this.test`? I didn't find the definition 😬

Reply:
There is a way how you can use current test context: https://mochajs.org/api/test.js.html
Agreed, documentation is pretty insufficient.

I have used the same strategy as here: https://github.com/googleapis/nodejs-compute/commit/2084e1c93016acb927e474adf488ca697de47a2f